benchmarks/benchAllocTasks.in.cpp

```diff
@@ -0,0 +1,63 @@
+#include <mc_rbdyn/RobotLoader.h>
+#include <mc_rbdyn/Robots.h>
+#include <mc_solver/QPSolver.h>
+#include <mc_tasks/MetaTaskLoader.h>
+#include <mc_tasks/SurfaceTransformTask.h>
+#include <mc_tasks/lipm_stabilizer/StabilizerTask.h>
+
+#include "benchmark/benchmark.h"
+
+class AllocTaskFixture : public benchmark::Fixture
+{
+public:
+  void SetUp(const ::benchmark::State &)
+  {
+    // XXX silence error output
+    std::cerr.rdbuf(nullptr);
```

Comment:
I'm not a big fan of this, but otherwise the benchmark output gets spammed with the warnings emitted when loading a robot (here missing force calibration files).